contracts/src/core/EigenDAServiceManagerStorage.sol

```diff
@@ -53,8 +53,8 @@ abstract contract EigenDAServiceManagerStorage is IEigenDAServiceManager {
     /// @notice mapping between the batchId to the hash of the metadata of the corresponding Batch
     mapping(uint32 => bytes32) public batchIdToBatchMetadataHash;
 
-    /// @notice address that is permissioned to confirm batches
-    address public batchConfirmer;
+    /// @notice mapping of addressed that are permissioned to confirm batches
+    mapping(address => bool) public isBatchConfirmer;
```

Comment:
this is changing in the same storage slot. will this break testnet deployment when upgrading?
I guess when upgrading we can just:

1. change the confirmer to the zero address 
2. upgrade
3. set batch confirmer